changelog.json

```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.8.3": [
+      "[Fixed] Rename a repo and then create and clone a repo with that repo's previous name now points to a new repo not the renamed one - #3855",
```

Comment:
Ugg.. That is a mouth full.... if I do say so myself..

Reply:
Cloning a repository with the same name as a previously cloned repository that has been renamed no longer points to previously cloned repository.

Is that better?